src/use-stepper.ts

```diff
@@ -212,6 +217,15 @@ function useStepper({
     const { ref, onBlur, onFocus, onChange, ...otherInputProps } = inputProps;
     return {
       ...otherInputProps,
+      inputMode: 'decimal',
+      pattern: '[0-9]*(.[0-9]+)?',
```

Comment:
I don't think we can assume `inputMode: 'decimal'` and `pattern: '[0-9]*(.[0-9]+)?'`. For example, the app which caused me to create this hook in the first place uses it for integers only - so I would want `inputMode: 'numeric'` and `pattern: '[0-9]*'`. Feels like these can be left to the consumer?

Reply:
Any thoughts on this one @wladiston? To me, it still feels like something the consumer needs to decide.